designer/client/src/javascripts/preview/select-sortable.js

```diff
@@ -0,0 +1,3 @@
+import { ListSortableQuestionElements } from '~/src/javascripts/preview/list-sortable'
```

Comment:
Missing ".js" from the import?

'~/src/javascripts/preview/list-sortable'
'~/src/javascripts/preview/list-sortable.js'